### ISSUE TITLE
Example for REDISQL.QUERY

### ISSUE THREAD

**Issue by shannonantony** [RedBeardLab/JRediSQL#1]
@siscia  can you give me an example for how to query from a table. I get error like

(error) ERR unknown command `REDISQL.QUERY`, with args beginning with: `SELECT * FROM foo`,

**Comment by siscia** [comment]
What command are you using?

**Comment by shannonantony** [comment]
I had created the DB and table and inserted the row into table foo1.
Then, I am doing like below command to run against the slave

List<Object> reply = rediSQL.query("DB", "SELECT * from foo1");

Basically, I am trying to insert the data into master and read from the slave. is that possible?

thanks

Jay

**Comment by siscia** [comment]
Hi!

Does the same command works against the master? 

Is the slave set up correctly? Does the query works from CLI?

Did you start the slave passing RediSQL as module to load?

Cheers,
Simone

**Comment by shannonantony** [comment]
Yes, everything works against the master. Query works against master using cli. I configured redisql.so file from the link https://github.com/RedBeardLab/rediSQL/releases/tag/v0.3.1.

I got the error while using the latest .so file available from github like GLIBC-2.18 not found. but i installed glibc 2.18 locally. but while running redis, it is not finding glibc. how do I configure that.

basic cli commands are not working from slave. Do i have to use the latest .so from github. please let me know the steps to build redisql.so from github source.

appreciate your help

thanks

jay

**Comment by shannonantony** [comment]
`

**Comment by siscia** [comment]
Easy! I am not following you!

So you have two nodes: master and slave but in the slave you are unable to run RediSQL. Is this correct?

To set the correct version of a dynamic library I suggest you to take your time to read this document here: http://man7.org/linux/man-pages/man8/ld.so.8.html

It is a great piece of documentation that explains in the details how a crucial part of Linux works. So it is worth to read! However it requires a little bit of attention!

Honestly my suggestion would be to use a docker container!
